app/models/items/voucher.rb

```diff
@@ -104,7 +104,7 @@ def item_description
 
   scope :open, -> { where(:checked_in => false).where(:showdate => nil) }
 
-  def unreserved? ; showdate_id.to_i.zero?  ;  end
+  def unreserved? ; showdate_id.to_i.zero? end
```

Comment:
this is technically ok but there should really be a semicolon before 'end' for clarity